project/assets/common.less

```diff
@@ -0,0 +1,40 @@
+footer {
+  background-color:  rgb(166, 166, 166);
+  color: rgb(77, 77, 77);
+  text-align: center;
+  font-family: roboto;
+  font-size: 20px;
+  font-weight: bold;
+  line-height: 105px;
+}
+
+button {
+  border: none;
+  background-color: rgb(51, 153, 255);
+  color: rgb(255, 255, 255);
+  font-family: roboto;
+  box-shadow: 0px 10px 35px -9px rgba(204,194,204,1);
```

Comment:
Alright, if you really, really want to! I was just asking out of curiosity, really. :)